openshift/templates/visual-verifier/visual-verifier-deploy.yaml

```diff
@@ -199,6 +236,11 @@ parameters:
     description: Used to group components together in the OpenShift console.
     required: true
     value: visual-verifier
+  - name: NAMESPACE_NAME
+    displayName: NameSpace name
+    description: name of the project namespace
+    required: true
+    value: myproject
```

Comment:
Similarly to the param files, please shift this close to `IMAGE_NAMESPACE`

Reply:
param moved